typescript/raven-tests.ts

```diff
@@ -1,8 +1,8 @@
-import Raven, {RavenOptions} from '..';
+import Raven = require('..');
 
 Raven.config('https://public@getsentry.com/1').install();
 
-var options: RavenOptions = {
+var options = {
```

Comment:
You cannot explicitly declare things as `RavenOptions` any more (can't be exported), but it will still be type-checked since it is passed as a parameter to `Raven.config`